hadoop-hdfs-project/hadoop-hdfs/src/main/java/io/hops/transaction/lock/LeaseLock.java

```diff
@@ -45,9 +47,17 @@
 
   @Override
   protected void acquire(TransactionLocks locks) throws IOException {
```

Comment:
I find the code of this function quite confusing, it would probably be good to break it in sub function or at least add some comments.

Reply:
New class will be better

Reply:
refactored